src/components/DeckMechanisms/index.js

```diff
@@ -158,10 +159,56 @@ export default class DeckMechanisms extends React.Component {
 
         // Remove the played card from the hand.
         newState.hand = state.hand.filter(cardId => cardId !== id)
-
+        
         if (options.discard) {
           return newState
         }
+        
+        // Turn one: Check if board is not full (by Rain of Frogs for example)
+        if (newState.turn === 1) {
+          const cardData = resolveCardForLevel(card)
+          // Any 3 mana card can be played since it would be the only one to be played
```

Comment:
Convention for comments: one empty line before, no empty line between the comment and the subject (and wrap at 80 characters column). ;)